src/huggingface_hub/README.md

```diff
@@ -13,9 +13,11 @@ files will be automatically downloaded again.
 `https://huggingface.co/julien-c/EsperBERTo-small/resolve/main/pytorch_model.bin`
 
 Parameters:
-- a repo id (e.g. a model id like `julien-c/EsperBERTo-small` i.e. a user or
+- a `repo_id` (e.g. a model id like `julien-c/EsperBERTo-small` i.e. a user or
   organization name and a repo name, separated by `/`)
-- a filename (like `pytorch_model.bin`)
+- a `filename` (like `pytorch_model.bin`)
+- an optional `subfolder`, equivalent to `EsperBERTo-small` (i.e. a repo name)
```

Comment:
I'm not sure I understand the explanation of `subfolder` here

Reply:
no it's a subfolder _inside_ the repo (so it's the same as `filename="foo/bar/pytorch_model.bin`)

we added this for a specific use case but it's not super useful param